README.md

```diff
@@ -1,55 +1,169 @@
-<mentoring url_name="goal_definition" followed_by="getting_feedback">
-    <html>
-        <p>What is your goal?</p>
-    </html>
+Install the requirements into the python virtual environment of your
+`edx-platform` installation by running the following command from the
+root folder:
 
-    <answer name="goal" />
-</mentoring>
+```bash
+$ pip install -r requirements.txt
 ```
 
-Second XBlock instance:
+Enabling in Studio
+------------------
 
-```xml
-<mentoring url_name="getting_feedback">
-    <html>
-        <p>The goal you entered was:</p>
-    </html>
-    <answer name="goal" read_only="true" />
-
-    <html>
-        <p>Ask feedback from friends about this goal - what did they think?</p>
-    </html>
-    <answer name="goal_feedback" />
-</mentoring>
-```
+You can enable the Mentoring XBlock in studio through the advanced
+settings.
+
+1. From the main page of a specific course, navigate to `Settings ->
+   Advanced Settings` from the top menu.
+2. Check for the `advanced_modules` policy key, and add `"mentoring"`
+   to the policy value list.
+3. Click the "Save changes" button.
+
+Usage
+-----
+
+When you add the `Mentoring` component to a course in the studio, the
+built-it editing tools guide you through the process of configuring the
+block and adding individual questions.
+
+### Mentoring modes
+
+There are 2 mentoring modes available:
+
+* *standard*: Traditional mentoring. All questions are displayed on the
+  page and submitted at the same time. The students get some tips and
+  feedback about their answers. This is the default mode.
+
+* *assessment*: Questions are displayed and submitted one by one. The
+  students don't get tips or feedback, but only know if their answer was
+  correct. Assessment mode comes with a default `max_attempts` of `2`.
+
+Below are some LMS screenshots of a mentoring block in assessment mode.
+
+Question before submitting an answer:
+
+![Assessment Step 1](https://raw.githubusercontent.com/edx-solutions/xblock-mentoring/1fb5e3ece6f34b6cf33c956a4fba1d7cbb7349a2/doc/img/assessment-1.png)
+
+Question after submitting the correct answer:
+
+![Assessment Step 2](https://raw.githubusercontent.com/edx-solutions/xblock-mentoring/1fb5e3ece6f34b6cf33c956a4fba1d7cbb7349a2/doc/img/assessment-2.png)
+
+Question after submitting a wrong answer:
+
+![Assessment Step 3](https://raw.githubusercontent.com/edx-solutions/xblock-mentoring/1fb5e3ece6f34b6cf33c956a4fba1d7cbb7349a2/doc/img/assessment-3.png)
+
+Score review and the "Try Again" button:
+
+![Assessment Step 4](https://raw.githubusercontent.com/edx-solutions/xblock-mentoring/1fb5e3ece6f34b6cf33c956a4fba1d7cbb7349a2/doc/img/assessment-4.png)
+
+### Free-form questions
+
+Free-form questions are represented by a "Long Answer" component. 
+
+Example screenshot before answering the question:
 
-### Self-assessment quizzes
+![Answer Initial](https://raw.githubusercontent.com/edx-solutions/xblock-mentoring/1fb5e3ece6f34b6cf33c956a4fba1d7cbb7349a2/doc/img/answer-1.png)
```

Comment:
You should just be able to do a relative path to the image file, and Github will grab it properly. Using these hashes like this means that if the file is updated later, it will not automatically update the README.